server/data/healthcheck.test.js

```diff
@@ -10,7 +10,6 @@ describe('service healthcheck', () => {
   })
 
   afterEach(() => {
-    nock.abortPendingRequests()
```

Comment:
Why remove this? You've got hanging `delay(1000)` otherwise.

Reply:
it said it was an invalid method name

Reply:
run npm install

Reply:
@nickmcmahon01 you need to run `npm i` as a lot of packages have been updated

@jonhindle I think nick actually fixed the issue that was causing the hang by writing mocks for the notificationService. This doesn't hang for me locally anymore

Reply:
I'll put it back in.

Reply:
I think it was more of a warning than a hang? Anything existing beyond the scope of its test could affect another test, so I'd still recommend cleaning up afterwards.